kinto-remote-settings/src/kinto_remote_settings/signer/listeners.py

```diff
@@ -60,10 +53,7 @@ def pick_resource_and_signer(request, resources, bucket_id, collection_id):
     return resource, signer
 
 
-def resource_group(resource, name, default):
-    group = resource.get(name, default)
-    # If review is configured per-bucket, the group patterns have to be replaced
-    # with the source collection id.
+def resource_group(resource, group):
```

Comment:
Do you think it's still worth it to have this `resource_group` function when now all we're doing is calling `<group>.format`?